src/clj/witan/send/send.clj

```diff
@@ -292,24 +257,15 @@
          (remove #(nil? (first %)))
          distinct)))
 
-(defworkflowfn prepare-send-inputs-1-0-0
+(defn build-input-datasets
+  "Build a map of the datasets to use for input"
+  [project-dir file-inputs schema-inputs]
+  (into {} (for [[k v] file-inputs]
+             [k (tu/csv-to-dataset (str project-dir "/" v) (k schema-inputs))])))
```

Comment:
Feel like `csv-to-dataset` should probably be moved to `utils.clj` rather than `test-utils.clj` if it's being used here

Reply:
We should address this in a further refactor I think.

Reply:
My REPL didn’t break.  But the result of the model run is a rather large map...

Reply:
My REPL didn’t break.  But the result of the model run is a rather large map...

Reply:
"We should address this in a further refactor I think." - fair enough

Do you see any reason to return the results to screen @mattford63 ?